securedrop/journalist_templates/edit_account.html

```diff
@@ -4,7 +4,7 @@
 {% if user %}
   {# Only admins may edit usernames and admin status #}
   <h1>{{ gettext('Edit user "{user}"').format(user=user.username) }}</h1>
-  <p><a href="/admin">{{ gettext('Back to admin interface') }}</a></p>
+  <p><a href="/admin">« {{ gettext('Back to admin interface') }}</a></p>
```

Comment:
This is great for consistency, but it's made me realize that all the places we use this as a navigational symbol probably look pretty odd in languages that use it as a quotation mark. Our links must look like they have more to say. :laughing: 

That's _another_ issue to file.

Reply:
I could remove the character on all admin pages. Let me know what you think.

Reply:
I've filed #5596 for that.